.github/workflows/build-test.yml

```diff
@@ -36,21 +35,17 @@ jobs:
 
        - name: Build sources
          run: |
-            $GITHUB_WORKSPACE/tools/iceoryx_build_test.sh build-strict build-all out-of-tree build-shared
+            sudo $GITHUB_WORKSPACE/tools/add_test_users.sh
+            $GITHUB_WORKSPACE/tools/iceoryx_build_test.sh build-strict build-all out-of-tree build-shared test-add-user
 
        - name: Build debian package
          run: |
             $GITHUB_WORKSPACE/tools/iceoryx_build_test.sh package
 
-       - name: Run timeless tests
+       - name: Run all tests
          run: |
            cd $GITHUB_WORKSPACE/build
-           make all_tests
-
-       - name: Run timing tests
-         run: |
-           cd $GITHUB_WORKSPACE/build
-           make timing_tests
+           tools/run_all_tests.sh all
```

Comment:
since the `run_all_tests.sh` script now takes a parameter to run all tests, shouldn't this be renamed to `run_tests.sh`?